docs/deploy/configurations.md

```diff
@@ -64,7 +64,8 @@ Currently, you can set only 1 namespace to watch in this flag. See [this Kuberne
 |aws-api-throttle                       | AWS Throttle Config             | [default value](#default-throttle-config ) | throttle settings for AWS APIs, format: serviceID1:operationRegex1=rate:burst,serviceID2:operationRegex2=rate:burst |
 |aws-max-retries                        | int                             | 10              | Maximum retries for AWS APIs |
 |aws-region                             | string                          | [instance metadata](#instance-metadata)    | AWS Region for the kubernetes cluster |
-|aws-vpc-id                             | string                          | [instance metadata](#instance-metadata)    | AWS VPC ID for the Kubernetes cluster |
+|aws-vpc-cache-duration                 | string                          | 5               | Length of time in minutes to cache VPC info |
```

Comment:
let's make aws-vpc-cache-duration to be `DurationVar`. it's default value will be something like `5m`

Reply:
sample: https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/main/pkg/config/runtime_config.go#L83